index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JIC Portfolio</title>
+
+	<link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700|Poppins" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:wght@100;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header class="Toolbar">
+    <a href="index.html" class="Mylogo Title2">JIC Software</a>
+    <div class="menu-pc">
+      <a class="item-menu-pc">Portfolio</a>
+      <a class="item-menu-pc">About</a>
+      <a class="item-menu-pc-group">
+      <a class="item-menu-pc">Contact</a>
+        <img class="item-menu-pc-icon" src="images/mail_icon.png" alt="email icon">
+      </a>
+    </div>
+    <nav class="menu-mobile">
+        <img class="burguer-menu" src="images/burguer_menu_icon.png" alt="burguer menu icon">
+    </nav>
+  </header>
+  <section class="main-frame">
+    <div class="content1">
+      <section class="Headline">
+        <h1 class="Primarytext Title1">Hey There.<br>I'm Jorge.</h1>
+        <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+        <p class="Descriptiontext Simpletext">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      </section>
+      <ul class="Socialmedia iconlist">
+        <li><a href=""><img class="socialmediaicon" src="images/icon1.png" alt="Git-Hub Icon"></a></li>
+        <li><a href=""><img class="socialmediaicon" src="images/icon3.png" alt="LinkedIn icon"></a></li>
+        <li><a href=""><img class="socialmediaicon" src="images/icon4.png" alt="Angelist icon"></a></li>
+        <li><a href=""><img class="socialmediaicon" src="images/icon2.png" alt="Twitter icon"></a></li>
+        <li><a href=""><img class="socialmediaicon" src="images/icon5.png" alt="Medium icon"></a></li>
+      </ul>
+    </div>
+    <div class="bgimg">
+      <img class="BackgroundImg" src="" alt="">
+    </div>
```

Comment:
Figma
![image](https://user-images.githubusercontent.com/78251927/227699995-a6cdb78d-746d-4e78-98e5-abb004e9b5cc.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/227700081-631e9bde-cb7c-4140-baf8-96db68cd4aa6.png)
The section does not have the proper background image 🙄


- [ ] As requested by the previous reviewer, kindly use a background image for the section. If an image is not part of the content (for aesthetic reasons) we use `CSS background image` not an `<img>` tag. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

- [ ] Also, after using the CSS background-image property for this section, kindly adjust the position of your background image to match the given template.

Reply:
Done. Thanks @abdulhamiid